.idea/modules/htsjdk.iml

```diff
@@ -39,5 +39,7 @@
         <SOURCES />
       </library>
     </orderEntry>
+    <orderEntry type="library" name="apache-ant-1.8.2-bzip2" level="application" />
+    <orderEntry type="library" name="testng-5.5-jdk15" level="application" />
```

Comment:
Ugh I didn't git add this! I'll need a reminder on how to get this file out of my commits